src-crg/source/index.html.md

```diff
@@ -0,0 +1,856 @@
+---
+title: CircleCI 2.1 Configuration Reference Guide
+
+language_tabs:
+  - shell
+
+search: true
+---
+
+# CircleCI 2.1 Configuration Reference Guide
+
+## Overview
+
+This document is a reference for the CircleCI 2.1 configuration keys used in the `config.yml` file. The presence of a `.circleci/config.yml` file in your CircleCI-authorized repository branch indicates you want to use the 2.1 infrastructure.
```

Comment:
The presence of the file only indicates you want to use CCI 2.x, 2.1 is indicated with the version key

Reply:
fixed